warehouse/models/intermediate/gtfs_quality/guidelines_checks/int_gtfs_quality__trip_planner_rt.sql

```diff
@@ -17,6 +17,7 @@ int_gtfs_quality__trip_planner_rt AS (
         CASE manual_check__gtfs_realtime_data_ingested_in_trip_planner
             WHEN 'Yes' THEN {{ guidelines_pass_status() }}
             WHEN 'No' THEN {{ guidelines_fail_status() }}
+            WHEN 'N/A - no fixed-route service' THEN {{ guidelines_na_check_status() }}
```

Comment:
I think this is fine for these purposes (like, within this PR) but ideally perhaps we might check that this aligns with that's in the warehouse/maybe there would be some kind of automated QC in Airtable to make sure that this isn't selected in error

Reply:
do you mean that we would actually query the distinct values from the airtable datatable and assign them here, rather than just writing them as text?

Reply:
Ope sorry should have been more specific -- I just meant that `no fixed-route service` should probably have an automated check to confirm it aligns with the new `fixed-route` checkbox / data structure (i.e., make sure that this value is not selected for a service that we elsewhere say has fixed-route service)